libraries/AP_GPS/AP_GPS.h

```diff
@@ -34,6 +34,7 @@
 #define GPS_MAX_RATE_MS 200 // maximum value of rate_ms (i.e. slowest update rate) is 5hz or 200ms
 #define GPS_UNKNOWN_DOP UINT16_MAX // set unknown DOP's to maximum value, which is also correct for MAVLink
 #define GPS_WORST_LAG_SEC 0.22f // worst lag value any GPS driver is expected to return, expressed in seconds
+#define GPS_MAX_DELTA_MS 245 // 200 ms (5Hz) + 50 ms buffer
```

Comment:
Comment doesn't match the value

Reply:
Yeah, I didn't want an actual 250ms buffet as that would have accepted a 4Hz GPS. Will update the comment none the less.